lib/Viroverse/Controller/Sample.pm

```diff
@@ -160,6 +162,164 @@ sub ice_cultures : Chained('page_base') PathPart('ice-cultures') Args(0) {
     $c->detach( $c->view("NG") );
 }
 
+sub isla_worksheet : Chained('page_base') PathPart('isla-worksheet') Args(0) {
+    my ($self, $c) = @_;
+
+    unless ($c->stash->{features}->{isla_worksheet}) {
+        return NotFound($c, "Feature disabled: ISLA worksheets");
+    }
+
+    my $sample_id = $c->model->sample_id;
+
+    my $buffer = IO::String->new;
+    my $xls = Excel::Writer::XLSX->new($buffer);
+    my $sheet = $xls->add_worksheet();
+    $sheet->set_column(0, 0, undef, undef, 1);
```

Comment:
Comment on what these `set_column` calls are doing, because positional args and the name of the method make it  kind of enigmatic.